docs/source/user/faq.rst

```diff
@@ -264,6 +264,12 @@ value, for example::
    import locale
    locale.setlocale(locale.LC_NUMERIC, 'C')
 
+How do I get Numba development builds?
+--------------------------------------
+
+Pre-release versions of Numba can be installed with conda::
```

Comment:
```suggestion
Pre-release versions of Numba can be installed with conda::

```

You are missing an empty line here without it, the documentation will be rendered as:

<img width="490" alt="Screen Shot 2020-10-19 at 11 40 30" src="https://user-images.githubusercontent.com/41563/96428460-136ad300-1200-11eb-96d1-055388eda6b7.png">

If you add the empty line as I suggested, it will render like:


<img width="490" alt="Screen Shot 2020-10-19 at 11 42 07" src="https://user-images.githubusercontent.com/41563/96428556-309fa180-1200-11eb-8f8d-168465dd285f.png">

Which is clearly, what you want in this case.